runtime/src/main/scala/org/apache/pekko/grpc/scaladsl/headers/headers.scala

```diff
@@ -111,3 +112,29 @@ object `Status-Message` extends ModeledCustomHeaderCompanion[`Status-Message`] {
   def findIn(headers: immutable.Seq[HttpHeader]): Option[String] =
     headers.collectFirst { case h if h.is(name) => h.value() }
 }
+
+final class `Trailer` private (values: immutable.Seq[String]) extends ModeledCustomHeader[`Trailer`] {
```

Comment:
OK to have it here for now, but perhaps also add a Pekko HTTP PR to model this header there?

Reply:
I've made it private to scaladsl

Reply:
I can take a look at opening a PR there this weekend